cmd/tailscale/cli/serve.go

```diff
@@ -66,10 +66,10 @@ EXAMPLES
   - To serve simple static text:
     $ tailscale serve https:8080 / text:"Hello, world!"
 
-  - To forward raw TCP packets to a local TCP server on port 5432:
-    $ tailscale serve tcp:2222 tcp://localhost:22
+  - To forward incoming TCP connections on port 2222 to a local TCP server on port 5432:
```

Comment:
this line's now 88 chars wide. From lines above, it looks like we're trying to be 80 char friendly. (some people configure their tiling window managers to have NxM terminals where each is 80 cols wide)